pkg/ingester/client/cortex.proto

```diff
@@ -56,9 +56,10 @@ message QueryResponse {
   repeated TimeSeries timeseries = 1 [(gogoproto.nullable) = false];
 }
 
-// QueryStreamResponse contains a batch of timeseries chunks.
+// QueryStreamResponse contains a batch of timeseries chunks or timeseries. Only one of these series will be populated.
 message QueryStreamResponse {
-  repeated TimeSeriesChunk timeseries = 1 [(gogoproto.nullable) = false];
+  repeated TimeSeriesChunk chunkseries = 1 [(gogoproto.nullable) = false];
```

Comment:
I'm personally not a big fan of this name. Have you considered `timeseries_chunk` which then gets generated into `TimeseriesChunk`?

Reply:
Hadn't, but I'm happy to change it to that if it's preferred.

Reply:
Don't have strong feelings, but to me it looks a bit more clear (being equal to the data type).